repo/connect.go

```diff
@@ -25,9 +25,9 @@ type ConnectOptions struct {
 var ErrRepositoryNotInitialized = errors.Errorf("repository not initialized in the provided storage")
 
 // Connect connects to the repository in the specified storage and persists the configuration and credentials in the file provided.
-func Connect(ctx context.Context, configFile string, st blob.Storage, password string, opt *ConnectOptions) error {
-	if opt == nil {
-		opt = &ConnectOptions{}
+func Connect(ctx context.Context, configFile string, st blob.Storage, password string, options *Options, connectOptions *ConnectOptions) error {
```

Comment:
Can we simply add Options as a field inside ConnectOptions?

Reply:
@jkowalski see if the recent commit is a fix.  Combining the two structures closer to the command line causes problems with the use case of each (one is for configuration only, the other is for general command-line flags) and visibility.

Reply:
Had a quick chat with Aaron, It is desirable to have a single options parser for CLI. Does embedding the structure into a section options struct for connect make them separate ?